### PR TITLE
CPU: Fixed include guards

### PR DESCRIPTION
**Pull request by zhuoshuguo**
Pull request created as per the issue  [#2623](https://github.com/RIOT-OS/RIOT/issues/2623#)

### CODE REVIEW

**Comment by zhuoshuguo** [comment]
@miri64 Sorry, I messed up with the old PR that was for "fix include guards" due to some mistaken operations, and I closed it...
Anyway, how about this one?

File: cpu/lpc11u34/include/LPC11Uxx.h [hunk 1]
```diff
@@ -662,5 +662,5 @@ typedef struct {
 #endif
 
 
-#endif  // __LPC11UXX_H__
+#endif  // LPC11UXX_H__
```
Comment (miri64) [inline cpu/lpc11u34/include/LPC11Uxx.h]: Can you, while your at it change this to C-style comments (`/* LPC11UXX_H__ */`)?

Comment (zhuoshuguo) [inline cpu/lpc11u34/include/LPC11Uxx.h]: Sure, no problem. :-)

File: cpu/kinetis_common/include/mcg.h [hunk 2]
```diff
@@ -130,7 +130,7 @@ int kinetis_mcg_set_mode(kinetis_mcg_mode_t mode);
 }
 #endif
 
-#endif /* __MCG_CPU_H */
+#endif /* KINETIS_CPU_USE_MCG */
 /** @} */
 
-#endif /* KINETIS_CPU_USE_MCG */
+#endif /* MCG_CPU_H */
```
Comment (zhuoshuguo) [inline cpu/kinetis_common/include/mcg.h]: @miri64 Martine, could you help check here. I though ```/* KINETIS_CPU_USE_MCG */``` and ```/* MCG_CPU_H */``` were misplaced in this file, and I made this revision. But I am not sure about it.

Comment (miri64) [inline cpu/kinetis_common/include/mcg.h]: Correct.

**Comment by miri64** [comment]
Please squash.

**Comment by zhuoshuguo** [comment]
squash done.

File: cpu/lpc2387/include/lpc2387.h [hunk 3]
```diff
@@ -151,4 +151,4 @@ extern "C" {
 }
 #endif
 
-#endif // __LPC2387_H
+#endif /* __LPC2387_H */
```
Comment (miri64) [inline cpu/lpc2387/include/lpc2387.h]: Mh, this one is still (or again?) prefixed with `__`. Please fix and squash immediately.

Comment (miri64) [inline cpu/lpc2387/include/lpc2387.h]: Immediate squashing can be done with `git commit --amend <file>` ;)

**Comment by miri64** [comment]
Apart from a minor issue: ACK, as soon as addressed and Murdock is green.

**Comment by zhuoshuguo** [comment]
Minor issue fixed.

**Comment by gebart** [comment]
Could you remove the trailing double underscores in the lpc files please? Other than that, (untested) ACK.

**Comment by gebart** [comment]
Sorry, I got some old mail notifications about this PR, I see that it's already merged

**Comment by miri64** [comment]
I actually prefer at least one, to differentiate them from normal macros, that's why I ACK'd regardless ;)

**Pull request merged**